src/controllers/events/delete_event.js

```diff
@@ -1,26 +1,45 @@
 const Request = require('request');
 const { eventsURL } = require('../../constants/urls.json');
+const jwt = require('jsonwebtoken');
 
 module.exports = (req, res) => {
-  const reqOptions = {
-    url: `${eventsURL}/${req.params.id}`,
-    method: 'DELETE'
-  };
-
-  Request(reqOptions, (error, response) => {
-    if (error) {
-      return res.render('error', {
-        statusCode: 500,
-        errorMessage: res.locals.localText.serverError
-      });
-    }
-    if (response.statusCode !== 204) {
-      return res.render('error', {
-        statusCode: 400,
-        errorMessage: res.locals.localText.badRequest
-      });
-    } else {
-      res.redirect(`/${req.params.lang}/events`);
-    }
-  });
+  if (req.cookies.token) {
```

Comment:
😬  sorry I should have seen this before: change to `(req.cookies && req.cookies.token)` to avoid any errors